lib/path.js

```diff
@@ -1522,24 +1523,7 @@ const posix = {
   },
 
 
-  format: function format(pathObject) {
-    if (pathObject === null || typeof pathObject !== 'object') {
-      throw new TypeError(
-          'Parameter "pathObject" must be an object, not ' + typeof pathObject
-      );
-    }
-
-    var dir = pathObject.dir || pathObject.root;
-    var base = pathObject.base ||
-               ((pathObject.name || '') + (pathObject.ext || ''));
-    if (!dir) {
-      return base;
-    }
-    if (dir === pathObject.root) {
-      return dir + base;
-    }
-    return dir + posix.sep + base;
-  },
+  format: (obj) => { return format('/', obj); },
```

Comment:
Ah, then...will go back to the declaration style used in the rest of the object...

...and done!